### ISSUE TITLE
Potential import collision: import path should be "knative.dev/serving", not "github.com/knative/serving" 

### ISSUE THREAD

**Issue by KateGo520** [fabric8io/kubernetes-client#2395]
## Background
I find that `knative.dev/serving` and `github.com/knative/serving` coexist in this repo：
https://github.com/fabric8io/kubernetes-client/blob/master/extensions/knative/generator/go.mod （Line 7 & 16）
```
github.com/knative/serving v0.13.2
knative.dev/serving v0.13.2 // indirect
```
That’s because the `knative/serving` has already renamed it’s import path from "github.com/knative/serving" to "knative.dev/serving". When you used the old path "github.com/knative/serving" to import `knative/serving`, go will reintroduces `knative/serving` through the import statements "import knative.dev/serving/…" in the go source file of `knative/serving`.

https://github.com/knative/serving/blob/v0.13.2/cmd/webhook/main.go#L39 
```
package main

import (
	// resource validation types
	autoscalingv1alpha1 "knative.dev/serving/pkg/apis/autoscaling/v1alpha1"
	net "knative.dev/serving/pkg/apis/networking/v1alpha1"
	"knative.dev/serving/pkg/apis/serving"
	v1 "knative.dev/serving/pkg/apis/serving/v1"
	"knative.dev/serving/pkg/apis/serving/v1alpha1"
	"knative.dev/serving/pkg/apis/serving/v1beta1"
	"knative.dev/serving/pkg/leaderelection"

	// config validation constructors
	tracingconfig "knative.dev/pkg/tracing/config"
	defaultconfig "knative.dev/serving/pkg/apis/config"
	autoscalerconfig "knative.dev/serving/pkg/autoscaler/config"
	"knative.dev/serving/pkg/deployment"
	"knative.dev/serving/pkg/gc"
	"knative.dev/serving/pkg/network"
	certconfig "knative.dev/serving/pkg/reconciler/certificate/config"
	istioconfig "knative.dev/serving/pkg/reconciler/ingress/config"
	domainconfig "knative.dev/serving/pkg/reconciler/route/config"
)…
```

The "knative.dev/serving" and "github.com/knative/serving" are the same repos. This will work in isolation, bring about potential risks and problems.

## Solution
Replace all the old import paths, change `"github.com/knative/serving"` to `"knative.dev/serving"`.
Where did you import it: https://github.com/fabric8io/kubernetes-client/search?q=github.com%2Fknative%2Fserving&unscoped_q=github.com%2Fknative%2Fserving

The same situation seems to be happening with "knative.dev/eventing" and "github.com/knative/eventing"

**Comment by KateGo520** [comment]
@rohanKanojia  @manusa  Could you help me review this issue? Thx :p

**Comment by rohanKanojia** [comment]
Thanks for your bug report. Actually this is the first time someone has reported about this. I think we're 
doing it here: 

https://github.com/fabric8io/kubernetes-client/blob/e8c47dc3ce9ecfc1567c4ed7b2d90f15b4cae8b6/extensions/knative/generator/cmd/generate/generate.go#L18-L35

**Comment by KateGo520** [comment]
Thank you for your reply! @rohanKanojia
Yes. You can update the import path. Replace all the old import paths, change "github.com/knative/serving" to "knative.dev/serving".

**Comment by rohanKanojia** [comment]
@KateGo520 : I've addressed this issue in #2470 . Could you please review https://github.com/fabric8io/kubernetes-client/pull/2470/files#diff-1f12452d1b29276896da60025e9cd04cR21 ?
